app/controllers/claims/pages/feedback_controller.rb

```diff
@@ -0,0 +1,26 @@
+class Claims::Pages::FeedbackController < Claims::ApplicationController
+  skip_before_action :authenticate_user!
+  before_action :skip_authorization
+  before_action :set_feedback, only: %i[new create]
+
+  def new; end
+
+  def create
+    if @feedback_form.valid?
+      # Send params to zendesk... if we use it
+      render :feedback_confirmation
```

Comment:
Since this is now a part of a feedback controller, should we name this view _just_ "confirmation"?